### PR TITLE
add secretory coil of eccrine sweat gland and def to duct of eccrine sweat gland

### LINKED ISSUE #2915: eccrine sweat gland structures
**Preferred term label:** secretory coil of eccrine sweat gland
**Definition** Tightly coiled tubule whose function is the production of sweat. The secretory tubule has three types of cells: clear, dark, and myoepithelial. The myoepithelial cells are contractile cells that provide structural support to the gland against hydrostatic pressure generated during sweat production. The secretory cells can be classified into clear cells and dark cells based on their affinities to basic dyes and granule contents. The clear cells are without secretory granules and are responsible for the secretion of primary sweat. The dark cells have abundant dark granules in the cytoplasm and are thought to regulate the function of the clear cells. 
(PMID:31608304; PMID:34395417)
**Parent term**
duct UBERON:0000058
‘part of’ some ‘eccrine sweat gland’ UBERON_0000423
**ORCID** https://orcid.org/0000-0002-0819-0473

Add to UBERON:0000423
**Definition** The main function of the duct of the eccrine sweet gland is the reabsorption of Na and Cl ions as sweat flows through the duct. It has two cell layers: basal and luminal. It is continuous with the secretory coil and goes towards the skin surface. The distal segment of the duct is relatively straight and connects with the acrosyringium in the epidermis. Most of the ion reabsorption occurs in the proximal part of the duct. 
(PMID:31608304)
**ORCID** https://orcid.org/0000-0002-0819-0473

### PR DESCRIPTION
**Pull request by paulacalipho**
fixes #2915

### CODE REVIEW

File: src/ontology/uberon-edit.obo [hunk 1]
```diff
@@ -182428,6 +182428,7 @@ intersection_of: part_of UBERON:0001044 ! saliva-secreting gland
 [Term]
 id: UBERON:0035073
 name: duct of eccrine sweat gland
+def: "The main function of the duct of the eccrine sweet gland is the reabsorption of Na and Cl ions as sweat flows through the duct. It has two cell layers: basal and luminal. It is continuous with the secretory coil and goes towards the skin surface. The distal segment of the duct is relatively straight and connects with the acrosyringium in the epidermis. Most of the ion reabsorption occurs in the proximal part of the duct." [PMID:31608304]
```
Comment (bvarner-ebi) [inline src/ontology/uberon-edit.obo:182431]: - While this seems correct, [it is noted in the issue](https://github.com/obophenotype/uberon/issues/2915#issue-1747169486) that this edit was intended for UBERON:0000423.

- Consider adding this first sentence: "A duct made up of simple tubular epithelium that is part of an eccrine sweat gland."

-  "... duct of the eccrine sweet gland... " --> "...  duct of the eccrine **sweat** gland... "

File: src/ontology/uberon-edit.obo [hunk 2]
```diff
@@ -224112,6 +224113,15 @@ relationship: part_of UBERON:0001769 ! iris
...
 property_value: dcterms-date "2023-06-06T08:06:04Z" xsd:dateTime
 
+[Term]
+id: UBERON:8480064
+name: secretory coil of eccrine sweat gland
+def: "Tightly coiled tubule whose function is the production of sweat. The secretory tubule has three types of cells: clear, dark, and myoepithelial. The myoepithelial cells are contractile cells that provide structural support to the gland against hydrostatic pressure generated during sweat production. The secretory cells can be classified into clear cells and dark cells based on their affinities to basic dyes and granule contents. The clear cells are without secretory granules and are responsible for the secretion of primary sweat. The dark cells have abundant dark granules in the cytoplasm and are thought to regulate the function of the clear cells." [PMID:31608304, PMID:34395417]
+is_a: UBERON:0000058 ! duct
```
Comment (bvarner-ebi) [inline src/ontology/uberon-edit.obo:224120]: Consider if 'epithelial tube' is accurate and more appropriate.

File: src/ontology/uberon-edit.obo [hunk 3]
```diff
@@ -224112,6 +224113,15 @@ relationship: part_of UBERON:0001769 ! iris
 property_value: curator_notes "In urodeles, lens regeneration proceeds through two steps. The first step involves the entire circumference of the iris pigmented epithelium. The second step is confined to the pigmented epithelial cells located along the dorsal rim of the iris and involves dorsal-specific activation of the Wnt signaling pathway." xsd:string
 property_value: dcterms-date "2023-06-06T08:06:04Z" xsd:dateTime
 
+[Term]
+id: UBERON:8480064
+name: secretory coil of eccrine sweat gland
+def: "Tightly coiled tubule whose function is the production of sweat. The secretory tubule has three types of cells: clear, dark, and myoepithelial. The myoepithelial cells are contractile cells that provide structural support to the gland against hydrostatic pressure generated during sweat production. The secretory cells can be classified into clear cells and dark cells based on their affinities to basic dyes and granule contents. The clear cells are without secretory granules and are responsible for the secretion of primary sweat. The dark cells have abundant dark granules in the cytoplasm and are thought to regulate the function of the clear cells." [PMID:31608304, PMID:34395417]
```
Comment (bvarner-ebi) [inline src/ontology/uberon-edit.obo:224119]: Consider moving **cell type** details to the CL terms.

**Review by bvarner-ebi — COMMENTED** [review]
Kindly see inline comments.

File: src/ontology/uberon-edit.obo (hunk 3 above)
Comment (bvarner-ebi) [reply src/ontology/uberon-edit.obo:224119]: Consider adding axioms to capture these cell types, e.g., 'has part' some 'clear cell of eccrine sweat gland'.

Comment (paulacalipho) [reply src/ontology/uberon-edit.obo:224119]: @bvarner-ebi I tried to add 'has part' some 'clear cell of eccrine sweat gland' as "SubClass Of" but I can't. Where these axioms must be added?

**Review by bvarner-ebi — APPROVED** [review]

**Pull request merged**